### ISSUE TITLE
Problem building 5.3.20

### ISSUE THREAD

**Issue by mattheath** [mattheath/puppet-php#5]
Error when building 5.3.20:

```
ld: library not found for -lltdl
```

Appears to be a dependency on libtool, which is not currently a requirement.

Full error:

```
ld: library not found for -lltdl
collect2: ld returned 1 exit status
make: *** [sapi/fpm/php-fpm] Error 1
```

**Comment by mattheath** [comment]
This stil occasionally appears even with libtool installed. Clearing the /vat/tmp/php-build folder appears to resolve

**Comment by mattheath** [comment]
Fixed by version `0.2.0` - php-build is now removed and a custom type and provider are used instead.
